`dist/bin/dotr` can't run with a custom `-classpath`

```
sbt dist-bootstrapped/pack
echo "object Main { def main(a: Array[String]): Unit = println(\"OK\") }" > Main.scala
./dist-bootstrapped/target/pack/bin/dotc -d out Main.scala
./dist-bootstrapped/target/pack/bin/dotr -classpath out Main
```

The last line will default to the repl instead running the application.

Note that `dist-bootstrapped/target/pack/bin/dotr` is generated from `dist/bin/dotr` when packing.

I changed the `dist/bin/dotr` to
```
if [ -z "$1" ]; then
    echo "Starting dotty REPL..."
    eval "$PROG_HOME/bin/dotc -repl"
elif [[ ${first_arg} == "-classpath" ]]; then
    eval exec "\"$JAVACMD\"" "$DEBUG" "-classpath \"$CLASS_PATH\"" $@
else
    eval "$PROG_HOME/bin/dotc -repl $@"
fi
```

But If i run your example I receive:
```
./dist-bootstrapped/target/pack/bin/dotr -classpath out Main
Exception in thread "main" java.lang.NoClassDefFoundError: scala/Predef$
	at Main$.main(Main.scala:1)
	at Main.main(Main.scala)
Caused by: java.lang.ClassNotFoundException: scala.Predef$
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 2 more
```

`CLASS_PATH` contains : `:/home/rsoeldner/work/dotty/dist-bootstrapped/target/pack/lib/dotty-library_0.5-0.5.0-bin-SNAPSHOT.jar:/home/rsoeldner/work/dotty/dist-bootstrapped/target/pack/lib/scala-library-2.12.4.jar` is this correct / what did i miss ?

--- a/project/scripts/sbtBootstrappedTests
+++ b/project/scripts/sbtBootstrappedTests
@@ -11,13 +11,14 @@
 echo "testing sbt dotc and dotr"
 mkdir out/scriptedtest0
 ./bin/dotc tests/pos/sbtDotrTest.scala -d out/scriptedtest0
+
 # FIXME #3477
-#./bin/dotr -classpath out/scriptedtest0 dotrtest.Test" > sbtdotr1.out
-#if grep -e "dotr test ok" sbtdotr1.out; then
-#    echo "output ok"
-#else
-#    exit -1
-#fi
+./bin/dotr -classpath out/scriptedtest0 dotrtest.Test" > sbtdotr1.out
+if grep -e "dotr test ok" sbtdotr1.out; then
+    echo "output ok"
+else
+    exit -1
+fi
 
 
 # check that `dotc` compiles and `dotr` runs it
@@ -26,10 +27,9 @@ mkdir out/scriptedtest1
 mkdir out/scriptedtest2
 ./bin/dotc tests/pos/sbtDotrTest.scala -d out/scriptedtest1/
 ./bin/dotc -from-tasty -classpath out/scriptedtest1/ -d out/scriptedtest2/ dotrtest.Test
-# FIXME #3477
-#./bin/dotr -classpath out/scriptedtest2/ dotrtest.Test" > sbtdotr2.out
-#if grep -e "dotr test ok" sbtdotr2.out; then
-#    echo "output ok"
-#else
-#    exit -1
-#fi
+./bin/dotr -classpath out/scriptedtest2/ dotrtest.Test" > sbtdotr2.out
+if grep -e "dotr test ok" sbtdotr2.out; then
+    echo "output ok"
+else
+    exit -1
+fi

--- a/project/scripts/sbtBootstrappedTests
+++ b/project/scripts/sbtBootstrappedTests
@@ -3,7 +3,6 @@
 # check that benchmarks can run
 ./project/scripts/sbt "dotty-bench-bootstrapped/jmh:run 1 1 tests/pos/alias.scala"
 
-
 # setup for `dotc`/`dotr` script tests
 ./project/scripts/sbt dist-bootstrapped/pack
 
@@ -12,7 +11,6 @@ echo "testing sbt dotc and dotr"
 mkdir out/scriptedtest0
 ./bin/dotc tests/pos/sbtDotrTest.scala -d out/scriptedtest0
 
-# FIXME #3477
 ./bin/dotr -classpath out/scriptedtest0 dotrtest.Test" > sbtdotr1.out
 if grep -e "dotr test ok" sbtdotr1.out; then
     echo "output ok"

